testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -497,125 +509,134 @@ def choosemeth(sel, meth, periodic):
 
         return sel
 
-    def _check_around(self, meth, periodic):
-        sel = Parser.parse('around 5.0 resid 1', self.u.atoms)
+    @pytest.mark.parametrize('u, meth, periodic', [
```

Comment:
If you change the `u` fixture above to a staticmethod, you don't have to parametrize around it here.

ie

```python
@staticmethod
@pytest.fixture
def u():

@pytest.mark.parametrize('meth, periodic', [])
def test_around(self, u, meth, periodic):
```

will work